generative_data_prep/data_prep/pipeline.py

```diff
@@ -31,16 +32,18 @@
 from generative_data_prep.data_prep import data_prep_main
 from generative_data_prep.processors.metrics import Metrics
 from generative_data_prep.utils import (
-    SEP_STR,
     BoundaryType,
     PackingConfig,
     balance_hdf5_files,
     execute_and_return_stdout,
     large_file_shuffle,
+    log_sep_str,
     verify_output_dir,
     verify_output_file,
 )
 
+logger = logging.getLogger("generative_data_prep_logger")
```

Comment:
use `__name__`

Reply:
I thought we wanted one logger so it is a "singleton" ? We can have separate level loggers for debugging in the future

Reply:
it is still the same logger, but it follows the packaging hierarchy implicitly if you do this.

Reply:
For context:

Reply:
Yeah, but that would be for a logger mean't to debug code. Here are the reasons I don't think we should do this

1. This will create a different logger for each file see this [source](https://docs.python.org/3/library/logging.html#:~:text=the%20final%20output.-,Logger%20Objects,to%20the%20same%20Logger%20object) 
2. This would use the root logger, not the generative_data_prep_logger, which I have prepared a config for.
3. We do not need access to the file name for this logger, since we do not want to print it out
4. We can always add another "debugger" logger that prints more information